### ISSUE TITLE
"% increased elemental damage" - implicit for spectres, and otherwise, are not in mod menu

### ISSUE THREAD

**Issue by qwak-adikt** [xyzz/acquisition#539]
Trying to search for "+% increased elemental damage" (not "with attacks", just the flat increase) but the mod is not there. It's the implicit on, e.g., Void Sceptres

**Comment by cr0s1s** [comment]
Loads of mods seem to be missing from the mod drop-down.
For example all of the evasion and armour modifiers are missing.

**Comment by qwak-adikt** [comment]
yes true, i hadn't even realized. no support for fractured either but a bit moot now

**Comment by RJonesUSC** [comment]
Any way to pull the new mods into the program so choosing them is an option?
